test-case/check-pause-resume.sh

```diff
@@ -17,8 +17,9 @@ set -e
 ##    no error happen for aplay/arecord
 ##
 
+TOPDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. && pwd)
```

Comment:
Please remove the extra signed-off-by line.

Reply:
Done. Great catch, I'm impressed :-)